ThreeBotPackages/tfgrid/threebot_provisioning/chatflows/threebot_reservation.py

```diff
@@ -0,0 +1,135 @@
+    reservation = j.sal.zosv2.reservation_create()
+
+    ip_version = bot.single_choice("choose your IP version", ips)
+    node_selected = j.sal.chatflow.nodes_get(1)[0]
+
+    # Encrypt AWS ID and AWS Secret to send it in secret env
+    aws_id_encrypt = j.sal.zosv2.container.encrypt_secret(node_selected.node_id, AWS_ID)
+    aws_secret_encrypt = j.sal.zosv2.container.encrypt_secret(node_selected.node_id, AWS_SECRET)
+
+    # Create network of reservation and add peers
+    if user_choice == "Restore":
+        hash_encrypt = j.sal.zosv2.container.encrypt_secret(node_selected.node_id, hash_restore)
+        env.update({"restore": "True"})
+        secret_env.update({"HASH": hash_encrypt})
+
+    reservation, config = j.sal.chatflow.network_configure(
+        bot, reservation, [node_selected], customer_tid=identity.id, ip_version=ip_version
+    )
+
+    ip_address = config["ip_addresses"][0]
+
+    backup = bot.single_choice("Auto-backup 3bot?", ["Yes", "No"])
+    if backup == "Yes":
+        password = bot.secret_ask("Backup password? Don't forget to save this password to be able to restore")
+        hash_backup = nacl.hash.blake2b(password.encode(), key=identity_pubkey.encode()).decode()
+        hash_encrypt = j.sal.zosv2.container.encrypt_secret(node_selected.node_id, hash_backup)
+        secret_env.update({"HASH": hash_encrypt})
+        env.update({"backup": "True", "FOLDER": folder_name})
+
+    env.update({"corex_password": password, "corex_user": user_corex, "pub_key": pub_key})
+    secret_env.update({"AWS_ID": aws_id_encrypt, "AWS_SECRET": aws_secret_encrypt})
+
+    container_flist = "https://hub.grid.tf/bola_nasr_1/threefoldtech-3bot-corex.flist"
+    entry_point = "/usr/bin/zinit init -d"
+    storage_url = "zdb://hub.grid.tf:9900"
+
+    # Add volume and create container schema
+    expiration = j.data.time.epoch + (60 * 60 * 24)
+    vol = j.sal.zosv2.volume.create(reservation, node_selected.node_id, size=8)
+    rid = j.sal.zosv2.reservation_register(reservation, expiration, customer_tid=identity.id)
+    # create container
+    # TODO:secret_env
+    cont = j.sal.zosv2.container.create(
+        reservation=reservation,
+        node_id=node_selected.node_id,
+        network_name=config["name"],
+        ip_address=ip_address,
+        flist=container_flist,
+        storage_url=storage_url,
+        env=env,
+        entrypoint=entry_point,
+        cpu=4,
+        memory=4096,
+        secret_env=secret_env,
+    )
+
+    j.sal.zosv2.volume.attach_existing(cont, vol, rid, "/sandbox/var")
+    expiration = j.data.time.epoch + (60 * 60 * 24)
+
+    resv_id = j.sal.zosv2.reservation_register(reservation, expiration, customer_tid=identity.id)
+
+    res = """# reservation sent. ID: {}
+        """.format(
+        resv_id
+    )
+    bot.md_show(res)
+
+    filename = "{}_{}.conf".format(name, resv_id)
+
+    res = """
+            # use the next template to configure the wg-quick config of your laptop:
```

Comment:
let's change this to 
"Use the following template to configure your wireguard connection. This will give you access to your 3bot.
Make sure you have wireguard (https://www.wireguard.com/install/) installed"